spec/classes/mod/info_spec.rb

```diff
@@ -140,7 +140,7 @@ def general_info_specs_apache24
   end
 
   context 'on a RedHat OS' do
-    include_examples 'RedHat 6'
+    include_examples 'RedHat 8'
 
     # Load the more generic tests for this context
     general_info_specs_apache22
```

Comment:
This is misleading. EL6 had apache 2.2 but EL8 has 2.4.

Reply:
Removed